site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,112 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Local Development Setup: Counters and Lists
```

Comment:
```suggestion
### Counters and Lists
```

Reply:
I haven't included it. Is it really necessary? 🤔